components/board/genericlinux/data.go

```diff
@@ -7,8 +7,7 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/pkg/errors"
-
+	"github.com/mkch/gpio"
```

Comment:
Does this still compile on a Mac? I wonder if we need to add a build tag to this file to be Linux-only, similar to the one at the top of gpio.go. The thorny part of this is that this file is used in the board.go files in lots of other directories (search for `genericlinux.GetGPIOBoardMappings`), meaning that _none_ of them could be built on non-Linux environments, meaning we might need to add build tags to lots of other files. Maybe there's a less invasive approach?